superset-frontend/stylesheets/less/cosmo/bootswatch.less

```diff
@@ -66,6 +66,10 @@
 
 // Buttons ====================================================================
 
+.btn {
+  text-transform: uppercase;
```

Comment:
Note: this also transforms datasource names in chart control panel:

<img src="https://user-images.githubusercontent.com/335541/85183787-de467800-b241-11ea-9c01-9f07850a7417.png" width="200">

which is probably not what we want.